setup.cfg

```diff
@@ -40,6 +40,7 @@ tests =
     invenio-db>=2.0.0,<3.0.0
     invenio-app>=2.1.0,<3.0.0
     invenio-indexer>=3.1.0,<4.0.0
+    invenio-users-resources>=7.0.0,<8.0.0
```

Comment:
is it still needed as a dependency since you are actually resolving the user in the builder (and the builder is defined in drafts-resources)?

Reply:
We concluded that this dependency can be moved to drafts-resources for now, where the `UserContext` implementation will take place. 

Later on we'll have to use a decoupled registry-style pattern to make sure we can easily define audit log actions across different core modules without ending up in circular dependencies.